roles/kafka_broker/tasks/kraft_rbac_mtls.yml

```diff
@@ -0,0 +1,13 @@
+---
```

Comment:
The name of the file doesn't sound very descriptive. 
Our general convention is to name the file to describe what's happening and not when the file is being called (that can be used as comments)

Reply:
makes sense, will "set_controller_principal.yml" be good?